.generator/templates/client.mustache

```diff
@@ -150,10 +159,23 @@ func NewPrivateKeyAuth(config PrivateKeyAuthConfig) *PrivateKeyAuth {
 	}
 }
 
-func (a *PrivateKeyAuth) Authorize() error {
+func (a *PrivateKeyAuth) Authorize(method, URL string) error {
 	accessToken, hasToken := a.tokenCache.Get(AccessTokenCacheKey)
 	if hasToken {
-		a.req.Header.Add("Authorization", "Bearer "+accessToken.(string))
+		nonce, hasNonce := a.tokenCache.Get(DpopAccessTokenNonce)
+		if !hasNonce {
+			a.req.Header.Add("Authorization", "Bearer "+accessToken.(string))
```

Comment:
Nit: Maybe we can do the same as in line 198 and get the token type from the cache instead of hardcoding?

Reply:
The risk of it is too many item getting cached and failure to retrieve token type despite everything else available is not something I would want. Obviously the argument for it will changed when there are more token type coming, but I think for now this should suffice